### DISCUSSION TITLE
Library Support

### DISCUSSION THREAD

**Discussion by pssolanki111** [pssolanki111/polygon#6] [Q&A]
Feel free to ask for help here from the community. Remeber to provide enough information and ensure to keep your credentials safe while sharing details.

**Comment by swingsio** [comment]
Hi I am trying to retrieve historical options data that includes Open Interest. Currently I only find historical price and OHLC data for these contracts.

**Comment by pssolanki111** [comment]
Hey @swingsio 
Although I did respond in the discord server, I don't think historical OI data is available. Best I've seen is the snapshot endpoint but that's also not historical.
